sketchcrapp.sh

```diff
@@ -106,15 +106,15 @@ EOF
 }
 # Help messages block
 usage() {
-  echo "Usage:"
-  echo "./sketchcrapp [-h] [-a] <applicationPath>"
-  echo "Supported versions: v63.1, v64.0, v65.1, v66.1, v67.1, [soon: v67.2]"
+  echo "[+] Usage:"
+  echo "[+] ./sketchcrapp [-h] [-a] <applicationPath>"
+  echo "[+] Supported versions: v63.1, v64.0, v65.1, v66.1, v67.1, v67.2"
```

Comment:
I would leave these 3 lines without `[+]`. (CLI Standard on stdin is to not show "fancy" things)